solarwinds_apm/distro.py

```diff
@@ -65,10 +71,42 @@ def _log_runtime(self):
         self._log_python_runtime()
         logger.info("OpenTelemetry %s/%s", sdk_version, inst_version)
 
+    def _get_token_from_service_key(self):
+        """Return token portion of service_key if set, else None"""
+        service_key = environ.get("SW_APM_SERVICE_KEY")
+        if not service_key:
+            logger.debug("Missing service key")
```

Comment:
I thought I would make this and others `debug` level because the missing/poorly-formatted key "only" means the default env var for `HEADERS` is set incorrectly. It is later in auto-instrumentation during ApmConfig initialization that APM Python is disabled due to bad key.